crates/zkvm/openvm/src/lib.rs

```diff
@@ -188,6 +188,11 @@ impl zkVM for EreOpenVM {
         Ok(public_values)
     }
 
+    type VerifyingKey = AggVerifyingKey;
+    fn get_verifying_key(&self) -> anyhow::Result<Self::VerifyingKey> {
+        Ok(self.agg_vk.clone())
```

Comment:
For OpenVM I think we should also include the `app_commit` as the `VerifyingKey`, because the `agg_vk` is universal, only the `app_commit` is related/generated from the program.

Reply:
Yeah `app_commit` seems closer to what we want, we should remove `agg_vkey` here.

>  It sounds like it can be reconstructed by anyone

Yes